python/ray/train/lightning/_lightning_utils.py

```diff
@@ -46,11 +50,9 @@ def node_rank(self) -> int:
         return session.get_node_rank()
 
     def set_world_size(self, size: int) -> None:
-        logger.warning("world_size setter is disabled in AIR LightningTrainer.")
```

Comment:
sorry to keep you back and forth, getting rid of the warning is ok, can you add a comment to explain a bit why this is disabled instead.
just say this is intentionally disabled, since global_rand() etc return data from Trainer session.